calebwagner/python-server
# DELETE methods added

#### Changes Made
1. Modified file `animals/request.py`, `customers/request.py`, `employees/request.py`, `locations/request.py` to include `DELETE method` functionality.
​
#### Steps to Review
1. Checkout this branch locally.
    ```
    git fetch --all
    git checkout cjw-ch5
    ```
2. Open a new Terminal tab (⌘T) and navigate to the server directory.
3. Test app functionality.
    > check for delete functions in each request.py package
4. View code file.
    > Confirm file modifications are present as indicated above.
